ert_shared/ensemble_evaluator/prefect_ensemble/storage_driver.py

```diff
@@ -27,6 +28,14 @@ def store(self, local_name, iens=None):
         shutil.copyfile(self._run_path / local_name, storage_uri)
         return storage_uri
 
+    def store_data(self, data, file_name, iens=None):
```

Comment:
This could potentially also be used by the `ert3.evaluator` as it currently writes the data to temp files and sends those in through the configuration...

Reply:
Sure it can also be used in `ert3.evaluator` I did not add it there just to keep changes to a minimum.

Reply:
That is fair! I was just thinking it would clean up the code... And these things are always easier to do while you are at it. But, you can also make an issue or a follow-up PR branching out from this one :)